.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
The issue is that without the install goal dependencies are not available to other modules.  I tried a local build as noted in earlier comments and it fails because crud cannot find common.  If there's some other goal that could be used other than install that's fine, but given the step in the travis config is 'install' this seemed the appropriate place.